packages/opentelemetry-core/test/trace/BasePlugin.test.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BasePlugin, NoopTracer, NoopLogger } from '../../src';
+import * as assert from 'assert';
+import * as path from 'path';
+
+const tracer = new NoopTracer();
+const logger = new NoopLogger();
+
+describe('BasePlugin', () => {
+  describe('internalFilesLoader', () => {
+    it('should load internally exported files', () => {
+      const testPackage = require('./fixtures/test-package');
+      const plugin = new TestPlugin();
+      assert.doesNotThrow(() => {
+        plugin.enable(testPackage, tracer, logger);
+      });
+
+      if (typeof process !== 'undefined' && process.release.name === 'node') {
```

Comment:
@draffensperger This is what I had in mind to "skip" this particular test in the browser

Reply:
OK, interesting. I'm alright with this for now, but could you add a `@todo` and a GitHub issue to create a separate package for this functionality (and link the issue into the comment)?

Reply:
Added todo & issue link